src/perform_step/low_order_rk_perform_step.jl

```diff
@@ -818,9 +818,9 @@ end
     # Hairer II, page 22
     ϱu, ϱd = zero(eltype(k7))^2, zero(eltype(g7))^2
     @. g6 = (g7 - g6)^2
```

Comment:
This should be changed I guess:
```suggestion
    @. g6 = g7 - g6
```